.github/workflows/build.yaml

```diff
@@ -47,3 +55,65 @@ jobs:
         with:
           name: release-${{ matrix.os }}
           path: "target/release/neolink*"
+  cross:
+    name: cross
+    runs-on: ubuntu-latest
+    container: "node:current-buster-slim"
+    strategy:
+      fail-fast: false
+      matrix:
+        # Everyone has a slightly different screwball naming scheme;
+        # Rust uses the target triple, GCC generally targets a family
+        # with a specific prefix, and dpkg's arch does its own thing.
+        include:
+          - arch: armhf
+            target: armv7-unknown-linux-gnueabihf
+            gcc: arm-linux-gnueabihf
+          - arch: arm64
+            target: aarch64-unknown-linux-gnu
+            gcc: aarch64-linux-gnu
+          - arch: i386
+            target: i686-unknown-linux-gnu
+            gcc: i686-linux-gnu
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install basic tools
+        run: |
+          apt-get update
+          apt-get install --assume-yes --no-install-recommends curl ca-certificates
+      - name: Install ${{ matrix.arch }} cross compiler and gstreamer
+        run: |
+          dpkg --add-architecture ${{ matrix.arch }}
+          apt-get update
+          apt-get install --assume-yes --no-install-recommends \
+          build-essential \
+          gcc-${{ matrix.gcc }} \
+          libgstrtspserver-1.0-dev:${{ matrix.arch }} \
+          libgstreamer1.0-dev:${{ matrix.arch }} \
+          libgtk2.0-dev:${{ matrix.arch }} \
+          libglib2.0-dev:${{ matrix.arch }}
+      - name: Install ${{ matrix.arch }} Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            override: true
+            target: ${{ matrix.target }}
+            profile: minimal
+      - name: Build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --all-features --target=${{ matrix.target }}
+        env:
+          CC_${{ matrix.triple }}: ${{ matrix.gcc }}-gcc
+          CXX_${{ matrix.triple }}: ${{ matrix.gcc }}-g++
```

Comment:
`matrix.triple` is undefined so this just becomes `CC_ = ...`

Reply:
Ha well I guess these lines aren't needed! 'swhat I get for trying this at 11:30pm